mate-settings-daemon/mate-settings-manager.c

```diff
@@ -157,11 +170,11 @@ static gboolean
 is_item_in_schema (char       **items,
                    const char  *item)
 {
-	while (*items) {
-	       if (g_strcmp0 (*items++, item) == 0)
-		       return TRUE;
-	}
-	return FALSE;
+        while (*items) {
+                if (g_strcmp0 (*items++, item) == 0)
+                        return TRUE;
+        }
+        return FALSE;
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.